internal/component/database_observability/mysql/collector/query_sample.go

```diff
@@ -264,24 +261,25 @@ func (c *QuerySample) fetchQuerySamples(ctx context.Context) error {
 			continue
 		}
 
-		cpuTime := float64(row.CPUTime) / 1e9
-		elapsedTime := float64(row.ElapsedTimePicoseconds.Int64) / 1e9
-
-		logMessage := fmt.Sprintf(
-			`schema="%s" digest="%s" digest_text="%s" rows_examined="%d" rows_sent="%d" rows_affected="%d" errors="%d" max_controlled_memory="%db" max_total_memory="%db" cpu_time="%fms" elapsed_time="%fms" elapsed_time_ms="%fms"`,
-			row.Schema.String,
-			row.Digest.String,
-			digestText,
-			row.RowsExamined,
-			row.RowsSent,
-			row.RowsAffected,
-			row.Errors,
-			row.MaxControlledMemory,
-			row.MaxTotalMemory,
-			cpuTime,
-			elapsedTime,
-			elapsedTime,
-		)
+		cpuTime := picosecondsToMilliseconds(row.CPUTime)
+		elapsedTime := picosecondsToMilliseconds(row.ElapsedTimePicoseconds.Float64)
+
+		logMessage :=
+			fmt.Sprintf(
+				`schema="%s" digest="%s" digest_text="%s" rows_examined="%d" rows_sent="%d" rows_affected="%d" errors="%d" max_controlled_memory="%db" max_total_memory="%db" cpu_time="%fms" elapsed_time="%fms" time="%dms" level="INFO"`,
```

Comment:
Was it intentional to remove `elapsed_time_ms` - it is actually used by the FE as `elapsed_time` get's removed during the label_value

Reply:
oops I'll put it back